bit/wallet.py

```diff
@@ -282,7 +282,8 @@ def create_transaction(
         :type leftover: ``str``
         :param combine: Whether or not Bit should use all available UTXOs to
                         make future transactions smaller and therefore reduce
-                        fees. By default Bit will consolidate UTXOs.
+                        fees. By default Bit will consolidate UTXOs. Note: When
+                        setting :param absolute_fee: this is ignored.
```

Comment:
This should be added to all `create_transaction` methods inside `bit/wallet.py` (for classes `PrivateKeyTestnet`, `MultiSig` and `MultiSigTestnet`).